test/core/test_capture.js

```diff
@@ -122,20 +256,96 @@ test('Capture before test - JSON', (t) => {
   });
 });
 
-test('Capture after test - JSON', (t) => {
-  const fn = path.resolve(__dirname, './scripts/after_test.json');
-  const script = require(fn);
-  runner(script).then(function (ee) {
+test('Capture and save to attribute of an Object in context.vars - JSON', (t) => {
```

Comment:
Might be staring at right at it and not seeing it but what's the difference between this test and the previous one? :D

Reply:
the first test captures a variable (`id`), the second captures a variable (`id`) as an object (`pet`) property

Reply:
so the second one makes the first one unnecessary then?